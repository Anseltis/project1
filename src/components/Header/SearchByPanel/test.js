import React from 'react'
import renderer from 'react-test-renderer'

import { SearchByPanel } from './index'

test('renders SearchByPanel properly', () => {
  const wrapper = renderer.create(<SearchByPanel />).toJSON()
  expect(wrapper).toMatchSnapshot()
})
