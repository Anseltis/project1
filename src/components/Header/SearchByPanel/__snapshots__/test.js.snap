// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SearchByPanel properly 1`] = `
<div>
  <span
    className="undefined__header"
  >
    Search by
  </span>
  <input
    className="undefined__title-input"
    defaultChecked={false}
    id="title"
    name="search-by"
    type="radio"
    value="title"
  />
  <label
    className="undefined__title-label"
    htmlFor="title"
  >
    Title
  </label>
  <input
    className="undefined__genre-input"
    defaultChecked={false}
    id="genre"
    name="search-by"
    type="radio"
    value="genre"
  />
  <label
    className="undefined__title-label"
    htmlFor="genre"
  >
    Genre
  </label>
</div>
`;
